dist/js/akit.component.chatbot-widget.js

```diff
@@ -123,10 +123,13 @@
                 $scope.$on('chatbotMessageReplyClicked', vm.sendReply);
 
                 vm.addToChat = function (message) {
-                    var newData = [].concat(vm.data, [
-                        Object.assign({}, message)
-                    ]);
-                    this.data = newData;
+                    if (message.type === "text" && message.message === "") {
```

Comment:
Aangezien hier niets tussen staat, is het misschien beter om er `if !(message.type === "text" && message.message === "") { ... }` van te maken?